src/apify/_actor.py

```diff
@@ -567,14 +566,20 @@ async def start(
 
         client = self.new_client(token=token) if token else self._apify_client
 
-        return await client.actor(actor_id).start(
-            run_input=run_input,
-            content_type=content_type,
-            build=build,
-            memory_mbytes=memory_mbytes,
-            timeout_secs=int(timeout.total_seconds()) if timeout is not None else None,
-            wait_for_finish=wait_for_finish,
-            webhooks=webhooks,
+        return ActorRun.model_validate(
+            await client.actor(actor_id).start(
+                run_input=run_input,
+                content_type=content_type,
+                build=build,
+                memory_mbytes=memory_mbytes,
+                timeout_secs=int(timeout.total_seconds()) if timeout is not None else None,
+                wait_for_finish=wait_for_finish,
+                webhooks=[
+                    hook.model_dump(by_alias=True, exclude_unset=True, exclude_defaults=True) for hook in webhooks
+                ]
+                if webhooks
+                else None,
+            )
```

Comment:
IMO these returns (not just this one) are quite complex commands and I'd split them into several smaller pieces. Not a request.